extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -360,7 +360,13 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     return this.mutateElement(() => {
       const {length} = this.elementsMetadata_[lightboxGroupId];
       this.maybeEnableMultipleItemControls_(length);
-      toggle(dev().assertElement(this.carousel_), true);
+      dev()
+        .assertElement(this.carousel_)
+        .getImpl()
+        .then((implementation) => {
+          implementation.unlayoutCallback();
```

Comment:
Could we add a comment explaining why we reset this first?

Reply:
Good idea, added a comment.